build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
it would be nice to know whether it is possible to limit the update of major component version, so it would be possible to avoid similar issues in the future. I'm thinking about the same logic, like in package.json for Node JS modules, where one can say "bla: ^1.0" and this will block the component "bla" from major version update (however, minor version updates are allowed)